### PR TITLE
# Add Simplified Chinese document guidelines.

### PR DESCRIPTION
**Pull request by Jinxiansen**

### CODE REVIEW

**Comment by Jinxiansen** [comment]
Yes, I think so too.
I very much look forward to the Swift server being able to become stronger under the leadership of the Vapor framework!
And more need Apple's support.

File: 4.0/docs/index.md [hunk 1]
```diff
@@ -2,6 +2,8 @@ This is the documentation for Vapor, a web framework for Swift.
 
 Vapor is the most used web framework for Swift. It provides a beautifully expressive and easy to use foundation for your next website or API.
 
+Read this in [简体中文](http://cn.docs.vapor.codes)
```
Comment (tanner0101) [inline 4.0/docs/index.md:5]: What about having this under the a "Languages" or "Translations" header? That way we have an obvious place to put additional translations. Something like:

```md
## Languages

- English [(docs.vapor.codes)](http://docs.vapor.codes)
- 简体中文 [(cn.docs.vapor.codes)](http://cn.docs.vapor.codes)
```

I think under the Getting Started header would make sense. We should have the same section in the chinese docs, too.

**Comment by Jinxiansen** [comment]
@tanner0101 

I put it on top of `Start`, maybe this is more eye-catching; if possible, I will sync this to the Chinese document.

**Comment by Jinxiansen** [comment]
@tanner0101 Ok, I have synchronized to the Chinese document.
http://cn.docs.vapor.codes

**Pull request merged**